htdocs/class/libraries/vendor/xoops/xmf/src/Xmf/Database/TableLoad.php

```diff
@@ -163,7 +163,7 @@ public static function extractRows($table, $criteria = null, $skipColumns = arra
         $rows = array();
         $result = $db->query($sql);
         if ($result) {
-            while ($row = $db->fetchArray($result)) {
+            while (false !== ($row = $db->fetchArray($result))) {
```

Comment:
Can you undo this change here, and submit it as a PR on XOOPS/xmf ?  Otherwise it will just be overwritten on the next build. That would help keep things in sync.